core/src/main/java/org/elasticsearch/cluster/metadata/MetaData.java

```diff
@@ -1552,7 +1552,7 @@ public static void toXContent(MetaData metaData, XContentBuilder builder, ToXCon
             builder.startObject("meta-data");
 
             builder.field("version", metaData.version());
-            builder.field("uuid", metaData.uuid);
+            builder.field("cluster_uuid", metaData.clusterUUID);
```

Comment:
> this is only used for internal serialization and is not exposed in the api. so I think we are good?

You are right, metadata is only exposed through cluster state that uses custom serialization which omits this.